src/api/payment-requests/payment-requests.md

```diff
@@ -64,26 +64,26 @@ version (documented on this page) and the "legacy" version (documented at
+| invoiceRef           | String             | A reference to an invoice for this payment request.                                                                                                                      |
+| redirectUrl          | String             | **Experimental** URL to redirect the user to after they pay or cancel the Payment Request. Must start with one of the `allowedRedirectUrls` for the [Merchant Config][]. |
+| externalRef          | String             | An external reference to the payment request                                                                                                                             |
+| terminalId           | String             | The software or logical id of the payment terminal.                                                                                                                      |
+| deviceId             | String             | The hardware id or serial number of the payment terminal.                                                                                                                |
+| operatorId           | String             | POS operator Id.                                                                                                                                                         |
+| createdByAccountId   | String             | Id of the [Centrapay Account][] creating the Payment Request.                                                                                                            |
+| createdByAccountName | String             | Name of the [Centrapay Account][] creating the Payment Request.                                                                                                          |
+| conditionsEnabled    | Boolean            | Flag to indicate that a merchant is able to accept [Payment Conditions](#payment-condition).                                                                             |
+| patronNotPresent     | Boolean            | Flag to indicate the patron is not physically present. This may affect payment conditions or available [Payment Options][].                                              |
+| cancellationReason   | String             | The reason that the payment request was cancelled. See [Cancellation Reasons](#cancellation-reasons) for possible values.                                                |
+| preAuth              | Boolean            | Flag to indicate the if the request is a pre authorization for supported [Asset Types][].                                                                                |
+| preAuthExpiresAt     | {% dt Timestamp %} | Pre authorization completions and releases will be accepted until this time.                                                                                             |
```

Comment:
We need error codes if you cant perform these actions, and they need to be documented below this table somewhere.

Reply:
I expect to see this new field in the examples that have preAuth: true too.

Reply:
Please update this when you're done:
https://www.notion.so/centrapay/The-API-Protocols-for-Preauth-d2f6800d164b47928d9fee0838b7b655

Reply:
Please update notion API internal documentation examples

Reply:
Change "pre authorization" to "Pre Auth" everywhere.